.travis.yml

```diff
@@ -0,0 +1,76 @@
+sudo: required
```

Comment:
Could you shorten the file so It will no have a redundant installation with respect to pysap?

Reply:
I have shortened it now and pushed locally. 
For now, we can just wait for it to pass. However, if things get too delayed and messy, I suggest we push this long version for now and later work on shortening.
In my opinion getting a Travis script up is at the highest priority. We can track the shortening in issue #24

Reply:
Done @LElgueddari , I have shortened it to minimum code in my opnion